tests/pkg_qdsa/tests/01-run.py

```diff
@@ -10,8 +10,10 @@
 from testrunner import run
 
 
+TIMEOUT = 12
```

Comment:
Maybe add a comment why this value is needed (see the script in tests/float as an example)

Reply:
addressed